lamindb/_file.py

```diff
@@ -65,8 +68,14 @@ def serialize(
             name = PurePath(key).name
         memory_rep = data
         suffix = infer_suffix(data, format)
-        # this is always local
-        filepath = lndb.settings.storage.cache_dir / name
+        # the following filepath is always local
+        if lndb.settings.storage.cache_dir is not None:
+            filepath = lndb.settings.storage.cache_dir / name
+        else:
+            # this should likely be added to lndb.settings.storage
+            cache_dir = Path(DIRS.user_cache_dir)
```

Comment:
Should this be cached / written directly into the instance storage, as it is only for local instances anyways.

Reply:
Yes, that would be reasonable. It could happen along similar lines to this: https://github.com/laminlabs/lndb-storage/blob/618a6b514c3937b5754be4596dbc47c4c74891f0/lndb_storage/object/_anndata.py#L16-L19

I just wasn't sure how the copy to storage part would be affected, because that's no longer going to be needed then.

I just wanted to move it out of the cwd for now.